content/en/network_monitoring/network_path/setup.md

```diff
@@ -325,6 +325,31 @@ Agent `v7.61+` is required.
 
 [3]: https://github.com/DataDog/datadog-agent/blob/2c8d60b901f81768f44a798444af43ae8d338843/pkg/config/config_template.yaml#L1731
 
+{{% /tab %}}
+{{% tab "Helm" %}}
+
+To enable Network Path with Kubernetes using Helm, add the below to your `values.yaml` file.
+Note: Helm chart v3.109.1+ **is required**. For more information, see the [Datadog Helm Chart][1] documentation.
+
+```yaml
+agents:
+  # [...]
+  containers:
+    # [...]
+    systemProbe:
+      env:
+        - name: DD_TRACEROUTE_ENABLED
+          value: "true"
+    processAgent:
+      env:
+        - name: DD_NETWORK_PATH_CONNECTIONS_MONITORING_ENABLED
+          value: "true"
+        ## [optional]
+        # - name: DD_NETWORK_PATH_COLLECTOR_WORKERS
+        #   value: "10"
+```
+[1]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L908
```

Comment:
I'm not sure I understand why this link goes to this specific line! Might be worth explaining what the user is supposed to do with this `traceroute` part of the docs? I'm not sure what it has to do with the note about the required version above, too.

Reply:
Ah! For that the **is required** is in bold in the section above as well so was trying to keep it consistent, I can remove it for both.

For the setup piece, they can enable Network Path with the envar `DD_TRACEROUTE_ENABLED` as the example shows, or use the helm chart example of `datadog.traceroute.enabled`, basically 2 ways to do the same thing. Most helm users are pretty familiar with either way but prefer to use the helm chart method if it's available (which it is now). Hope that helps!

https://datadoghq.atlassian.net/wiki/spaces/NET/pages/3944745291/Network+Path+-+FAQ#How-to-setup-Network-Path-using-Datadog-Helm-Chart-in-k8s?

Reply:
fixed this to make it more clear and provide the actual helm values instead of envars, and linking to the helm chart README instead of individual lines